autoscale/src/main/java/com/sequenceiq/periscope/controller/AlertController.java

```diff
@@ -229,16 +228,6 @@ private void validateLoadAlert(Long clusterId, Optional<Long> alertId, LoadAlert
                 () -> {
                     validateAccountEntitlement(cluster);
                     validateSupportedHostGroup(cluster, json.getScalingPolicy().getHostGroup(), AlertType.LOAD);
-                    String requestHostGroup = json.getScalingPolicy().getHostGroup();
-                    cluster.getLoadAlerts().stream().map(LoadAlert::getScalingPolicy).map(ScalingPolicy::getHostGroup)
-                            .filter(hostGroup -> hostGroup.equalsIgnoreCase(requestHostGroup)).findAny()
-                            .ifPresent(hostGroup -> {
-                                throw new BadRequestException(messagesService
-                                        .getMessage(MessageCode.LOAD_CONFIG_ALREADY_DEFINED, List.of(cluster.getStackName(), requestHostGroup)));
-                            });
-                    clusterProxyConfigurationService.getClusterProxyUrl()
-                            .orElseThrow(() ->  new BadRequestException(
-                                    messagesService.getMessage(MessageCode.CLUSTER_PROXY_NOT_CONFIGURED, List.of(cluster.getStackName()))));
                 });
     }
 
```

Comment:
ClusterProxy is not a user controlled config, hence removed this user facing validation and also all clusters are registered with clusterproxy.  
Multiple load-alerts in a request is already validated in API layer, hence removed duplicate check.

Reply:
Is ClusterProxy always on? (Is there any way to disable it, via the CLI for example?) Cloudbreak still supports DIRECT and CLUSTER_PROXY (Tunnel.java)

Reply:
Yes it is always on. The CLI tunnel just decides how CB talks to cluster but cluster proxy registration is always done. Also cluster access via IP i.e DIRECT is not always guaranteed.